docs/installation.md

```diff
@@ -250,11 +235,11 @@ To configure delayed termination on the NGF Pod when the deployment method is He
 1. Uninstall the NGINX Gateway Fabric:
 
    ```shell
-   kubectl delete -f deploy/manifests/nginx-gateway.yaml
+   kubectl delete -f https://github.com/nginxinc/nginx-gateway-fabric/releases/download/v1.0.0/nginx-gateway.yaml
    ```
 
    ```shell
-   kubectl delete -f deploy/manifests/crds
+   kubectl delete -f https://github.com/nginxinc/nginx-gateway-fabric/releases/download/v1.0.0/crds.yaml
    ```
 
```

Comment:
Just saw the upgrade instructions also reference the repo files. Do those need to be updated? Or do we need to wait until the next release?

Reply:
I'll update those too, could apply to a user coming from 0.6.0